.codesandbox/templates/next/util/useCanvas.ts

```diff
@@ -0,0 +1,56 @@
+import { fabric } from "fabric";
+import { useCallback, useEffect, useRef } from "react";
+
+const DEV_MODE = process.env.NODE_ENV === "development";
+
+export function useCanvas(
```

Comment:
we may want to simplify this method
I took it from existing code

Reply:
for example `saveState` isn't so good because if code changes in the `init` block it doesn't take effect on hot reload, but only on hard reload

So maybe I should remove it entirely

It will be bad if the sandbox raises bugs because it is supposed to be a bug repro....